fastmri/data/mri_data.py

```diff
@@ -358,3 +360,153 @@ def __getitem__(self, i: int):
             sample = self.transform(kspace, mask, target, attrs, fname.name, dataslice)
 
         return sample
+
+
+class AnnotatedSliceDataset(SliceDataset):
+    def __init__(
+            self,
+            root: Union[str, Path, os.PathLike],
+            challenge: str,
+            mri_type: str,
+            multiple_annotation_policy: str,
+            transform: Optional[Callable] = None,
+            use_dataset_cache: bool = False,
+            sample_rate: Optional[float] = None,
+            volume_sample_rate: Optional[float] = None,
+            dataset_cache_file: Union[str, Path, os.PathLike] = "dataset_cache.pkl",
+            num_cols: Optional[Tuple[int]] = None,
+            annotation_version: Optional[str] = None,
+    ):
+        """
+        Args:
+            root: Path to the dataset.
+            challenge: "singlecoil" or "multicoil" depending on which challenge
+                to use.
+            mri_type: 'knee' or 'brain' depending on which dataset to use.
+            multiple_annotation_policy: 'first', 'random' or 'all'. 
+                If 'first', then only use the first annotation.
+                If 'random', then pick an annotation at random.
+                If 'all' then two or more copies of the same slice for each annotation
+                will be extended.
+            transform: Optional; A callable object that pre-processes the raw
+                data into appropriate form. The transform function should take
+                'kspace', 'target', 'attributes', 'filename', and 'slice' as
+                inputs. 'target' may be null for test data.
+            use_dataset_cache: Whether to cache dataset metadata. This is very
+                useful for large datasets like the brain data.
+            sample_rate: Optional; A float between 0 and 1. This controls what fraction
```

Comment:
No need to include `Optional;`.

Reply:
I just copy all args from `SliceDataset` and add new args. So should I delete all optional args?

Reply:
Okay in that case I think you can leave it. I think eventually we would like to remove all "Optional" from docstrings but I'll take care of that later.